UX-Guide-Metadata/draft/techniques/onix-metadata/index.html

```diff
@@ -173,6 +173,32 @@ <h3>Check for Node</h3>
 					<li><b>ELSE</b> return <code>False</code>.</li>
 				</ol>
 			</section>
+			<section id="join-array-to-comma-list">
```

Comment:
If you join an array with only one element, you get that element back, so only an array with 2 or more hazards will have commas. If you then replace the last comma with " and" you'll get the same result as this function, as it again won't alter the single-hazard case. I'd expect most languages have a "join" function, so this might be more than is needed.